### PR TITLE
Fix/typos links

### LINKED ISSUE #347: Several Links are broken in the docs
Several links are broken across the following sections:

- Installation
- Compose
- Collaborate
- Update
- Workspace
- Components
- Dependencies
- Scope
- Envs
- Dev Services

The links should be revisited and fixed.

### PR DESCRIPTION
**Pull request by jonatankBIT**
closed #347 
Several Links in the doc had been fixed across the following sections:

- Installation
- Compose
- Collaborate
- Update
- Workspace
- Components
- Dependencies
- Scope
- Envs
- Dev services

There are still sections to be reviewed (Apps, Packages, Frameworks, Extending bit, Config, Reference)

### CODE REVIEW

**Comment by ranm8** [comment]
You rock Jonathan! @giteden can you please review and merge!

**Review by giteden — APPROVED** [review]

File: workspace/content/workspace-link/workspace-link.mdx [hunk 1]
```diff
@@ -29,7 +29,7 @@ Workspace links are generated in the `node_modules` directory and are composed o
           ├── ...
 ```
 
-Workspace Component Links are generated in the `node_modules` directory using `bit link`.
+Workspace Componnent Link are generated in the `node_modules` directory using `bit link`.
```
Comment (KutnerUri) [inline workspace/content/workspace-link/workspace-link.mdx:32]: typo - the previous was correct. This is happening more than one, please check each place

**Review by KutnerUri — CHANGES_REQUESTED** [review]

File: docs/content/getting-started/create-workspace/create-workspace.mdx [hunk 2]
```diff
@@ -36,7 +35,7 @@ Congrats! You have created a Bit Workspace. You should get a similarly structure
 ├── .gitignore
 ├── README.md
 ├── pnpm-lock.yaml --> chosen package manager lock file
-├── node_modules --> includes external workspace modules and Workspace Component Links.
+├── node_modules --> includes external workspace modules and Workspace Componnent Link.
```
Comment (EllaStahl) [inline docs/content/getting-started/create-workspace/create-workspace.mdx:38]: Please fix all "componnent" to "component" (I found 7 with search)

**Comment by NitsanCohen770** [comment]
@jonatankBIT there are a lot of conflicts
can you please resolve them asap?
we are moving bit.dev to lanes
thanks!

File: workspace/content/workspace-link/workspace-link.mdx (hunk 1 above)
Comment (jonatankBIT) [reply workspace/content/workspace-link/workspace-link.mdx:32]: Fixed

File: docs/content/getting-started/create-workspace/create-workspace.mdx (hunk 2 above)
Comment (jonatankBIT) [reply docs/content/getting-started/create-workspace/create-workspace.mdx]: Fixed, thanks!

**Comment by jonatankBIT** [comment]
> @jonatankBIT there are a lot of conflicts can you please resolve them asap? we are moving bit.dev to lanes thanks!

working on that

**Comment by jonatankBIT** [comment]
@KutnerUri @NitsanCohen770 @EllaStahl Updated requested changes.

File: compilation/content/compiling-during-build/compiling-during-build.mdx [hunk 3]
```diff
@@ -9,11 +9,11 @@ This means Source Maps <!--TODO [Source Maps](/) --> are generated and component
 
 :::note
 
-For type resolution in the Workspace we rely on the [Workspace Component Link](/workspace/structure/component-link) which symlinks the source code from within the link to the corresponding [Component Directory](/workspace/structure/component-directory).
+For type resolution in the Workspace we rely on the [Workspace Component Links](/workspace/structure/component-links) which symlinks the source code from within the link to the corresponding [Component Directory](/workspace/structure/component-directory).
```
Comment (KutnerUri) [inline compilation/content/compiling-during-build/compiling-during-build.mdx:12]: @NitsanCohen770 
verify this type of links, I think they need to start with `/docs/...`

**Review by KutnerUri — APPROVED** [review]

**Pull request merged**